test/integration_test/migration_test.go

```diff
@@ -442,6 +443,19 @@ func migrationLabelSelectorTest(t *testing.T) {
 	)
 }
 
+func migrationLabelExcludeSelectorTest(t *testing.T) {
+	triggerMigrationTest(
+		t,
+		"migration-label-exclude-selector-test",
+		"cassandra",
+		[]string{"mysql-1-pvc"},
```

Comment:
How are we verifying if mysql app is not getting migrated?

Reply:
I am just following the existing framework.

appKey = cassandra and additionalAppKeys = mysql, migrateAllAppsExpected = false, startAppsOnMigration = true

func() validateAndDestroyMigration 
: allAppsCtx -> Cassandra (Cassandra migrated)
if (migrationSuccessExpected) -> true 
 if !migrateAllAppsExpected -> true
   require.Error(t, err, "All apps shouldn't have been migrated")
This means if mysql got migrated and scheduled we would report and error and test would fail.

Reply:
migrationLabelSelectorTest is implemented in a similar way.